README.md

```
@@ -1,5 +1,41 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/jeanmolossi/MaiGo.svg)](https://pkg.go.dev/github.com/jeanmolossi/MaiGo)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)